### PR TITLE
Clean pro finder.server edit user profile endpoint

### PR DESCRIPTION
**Pull request by s1uns**

### CODE REVIEW

File: CleanProFinder/CleanProFinder.Server/Controllers/ProfileController.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <remarks>
+        /// If the operation is successful, it will return a UserProfileDto.
+        /// If there is a bad request, it will return an ErrorDto.
+        /// </remarks>
+        /// <returns>An IActionResult representing the result of the operation.</returns>
+        [HttpPost("service-user/edit")]
+        [ProducesResponseType(typeof(UserProfileDto), 200)]
+        [ProducesResponseType(typeof(ErrorDto), 400)]
+        public async Task<IActionResult> SignUpServiceUser(EditUserProfileCommand request,
```
Comment (Wednumi) [inline CleanProFinder/CleanProFinder.Server/Controllers/ProfileController.cs:38]: you forgot to change method name

File: CleanProFinder/CleanProFinder.Server/Controllers/ProfileController.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        /// Update a service user profile.
+        /// </summary>
+        /// <param name="request">The request to update a service user profile.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <remarks>
+        /// If the operation is successful, it will return a UserProfileDto.
+        /// If there is a bad request, it will return an ErrorDto.
+        /// </remarks>
+        /// <returns>An IActionResult representing the result of the operation.</returns>
+        [HttpPost("service-user/edit")]
```
Comment (Wednumi) [inline CleanProFinder/CleanProFinder.Server/Controllers/ProfileController.cs:35]: you need to restrict this request for authorized users with role of service user

Comment (s1uns) [reply CleanProFinder/CleanProFinder.Server/Controllers/ProfileController.cs:37]: fixed

File: CleanProFinder/CleanProFinder.Server/Controllers/ProfileController.cs (hunk 1 above)
Comment (s1uns) [reply CleanProFinder/CleanProFinder.Server/Controllers/ProfileController.cs]: fixed

**Review by Wednumi — APPROVED** [review]

**Pull request merged**